ambry-clustermap/src/main/java/com.github.ambry.clustermap/HelixClusterManager.java

```diff
@@ -128,6 +130,7 @@ public void run() {
       initializationException.compareAndSet(null, e);
     }
     if (initializationException.get() == null) {
+      sealedStateChangeCounter.incrementAndGet();
```

Comment:
Would prefer if the increment is done in a single place - helps with understanding quicker.